cmd/finalize-bundle/main.go

```diff
@@ -9,25 +9,25 @@ import (
 )
 
 func main() {
-	bundlePath, manifestPath, manifestDestinationPath, err := verifyCommandLineArgs(os.Args)
+	bundlePath, bundleDestinationPath, manifestPath, manifestDestinationPath, err := verifyCommandLineArgs(os.Args)
```

Comment:
do we still need the bundleDestinationPath?

Reply:
Yes, this path points to build dir where the newly generated file will be put.